<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>921</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lblPath">
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditFolder">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetFilesToAnalyze">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="toolBtnOpenFiles">
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/open.png</normaloff>:/open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolBtnClearFileList">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/edit_clear.png</normaloff>:/edit_clear.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolBtnParse">
           <property name="text">
            <string>Parse</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/arrow.jpg</normaloff>:/arrow.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QMdiArea" name="mdiArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolBtnSaveResultAs">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/save.png</normaloff>:/save.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolBtnOpenResult">
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/open.png</normaloff>:/open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolBtnClearResults">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/edit_clear.png</normaloff>:/edit_clear.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
